src/intl/en.json

```diff
@@ -1398,5 +1398,26 @@
   "page-dapps-editors-choice-foundation": "Invest in culture. Buy, trade, and sell unique digital artwork and fashion from some incredible artists, musicians, and  brands.",
   "page-dapps-editors-choice-pooltogether": "Buy a ticket for the no-loss lottery. Each week, the interest generated from the entire ticket pool is sent to one lucky winner. Get your money back whenever you like.",
   "page-layout-banner-staking-1": "Staking has arrived! If you're looking to stake your ETH",
-  "page-layout-banner-staking-2": "confirm the deposit contract address"
+  "page-layout-banner-staking-2": "confirm the deposit contract address",
+  "page-assetdownload-artist": "Artist:",
+  "page-assetdownload-download": "Download",
+  "page-bugbountypoints-exchange": "Points Exchange",
+  "page-bugbountypoints-point": "1 point",
+  "page-bugbountypoints-error": "Error loading data... please refresh.",
+  "page-bugbountypoints-loading": "Loading data...",
+  "page-bugbountypoints-usd": "2 USD",
+  "page-bugbountypoints-payout-desc": " The Ethereum Foundation will pay out the value of USD in ETH or DAI.",
+  "page-bugbountypoints-rights-desc": "The Ethereum Foundation reserves the right to change this without prior notice.",
+  "page-calltocontribute-title": "Help us with this page",
+  "page-calltocontribute-desc-1": "If you're an expert on the topic and want to contribute, edit this page and sprinkle it with your wisdom.",
+  "page-calltocontribute-desc-2": "You'll be credited and you'll be helping the Ethereum community!",
+  "page-calltocontribute-desc-3": "Use this flexible",
+  "page-calltocontribute-link": "documentation template",
+  "page-calltocontribute-desc-4": "Questions? Ask us in the #content channel on our",
+  "page-calltocontribute-link-2": "Discord server",
+  "page-calltocontribute-span": "Edit page",
+  "page-codeblock-copy": "Copy",
+  "page-codeblock-copied": "Copied",
+  "page-docsnav-previous": "PREVIOUS",
```

Comment:
We should probably uppercase these strings in the component w/ CSS vs. storing uppercase here.